text/0000-selecting-default-java-version.md

```diff
@@ -0,0 +1,34 @@
+# Selecting the next default Java Version for Paketo Buildpacks
+
+## Summary
+
+This RFC introduces a process to select the next default version of the Paketo Java Buildpack.
+
+## Motivation
+
+Currently, there is no process how the default version of Java is picked in the Paketo world. As of today (January 2023), the default still is Java 11 (released September 2018) although Java 17 (released September 2021) is already available.
+
+## Detailed Explanation
+
+The default should be changed according to a defined process rather than at some undefined point in time to some undefined version. To do so, the default should be changed once the latest releases LTS version of Java is at least one year old. 
+
+## Rationale and Alternatives
+
+- Do nothing, update default manually from time to time
+- Always use the latest LTS immediately
+  - Could break people immediately
+- This RFC
+  - Gives room to adapt to breaking changes
+
+## Implementation
```

Comment:
I don't think we need a ton of action here, but I would like to have the implementation include documenting this process on paketo.io/docs.

I would also like to see a description of the notification channels/process that should be used to alert folks of the upcoming change.

Like at 3 months prior to the change, we'll post on Slack & open a Github issue to upgrade (maybe we also embed a message in the buildpack output?).  At 1 month prior, we'll post a Paketo blog entry on the topic. Then after the change we'll post another blog post when the default change has been released.

I'm not saying it has to be exactly that, but I think we should document what folks can expect in terms of notifications also.

Reply:
I added a sentence about documenting the process. I like your proposal, but I'm not sure about adding it to the build output.